### PR TITLE
Add Task06 and Task07 from Assignment4

### PR DESCRIPTION
**Pull request by laurandara**

### CODE REVIEW

File: Assignment4/laurandara-23A276/task07.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+NS = Namespace("http://somewhere#")
+
+q1 = prepareQuery('''
+  SELECT ?subclass
+  WHERE
+  {
+    ?subclass rdfs:subClassOf ns:LivingThing.
+  }
+  ''',
```
Comment (anaigmo) [inline Assignment4/laurandara-23A276/task07.py:35]: missing recursive sublcass of relationships

File: Assignment4/laurandara-23A276/task07.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+"""**TASK 7.4:  List the name of the persons who know Rocky**"""
+
+q4 = prepareQuery('''
+    SELECT DISTINCT ?person
+    WHERE
+    {
+        {
+          ?person foaf:knows ns:RockySmith .
+        }
+    }
```
Comment (anaigmo) [inline Assignment4/laurandara-23A276/task07.py:89]: one set of curly brackets is enough, and the name is asked for, not the iri!

File: Assignment4/laurandara-23A276/task07.py
Comment (anaigmo) [inline Assignment4/laurandara-23A276/task07.py:1]: missing rdflib alternatives for all exercise in this task

**Comment by laurandara** [comment]
Good afternoon, the errors mentioned have been fixed!

**Comment by anaigmo** [comment]
Submitted and fixed, no more PRs allowed

**Pull request merged**